### PR TITLE
link added to edit button and conditional rendering by id

### PR DESCRIPTION
**Pull request by mlync87**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Auenc) [inline .vscode/settings.json:1]: This is a personal setting file, we shouldn't commit it

File: button.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
+<!-- HTML !-->
+<button class="button-4" role="button">Button 4</button>
```
Comment (Auenc) [inline button.css:2]: Why do we have HTML in this css file?

File: src/components/profile/ProfilePage.js [hunk 3]
```diff
@@ -1,61 +1,73 @@
...
-import { useParams } from 'react-router-dom'
+import { useParams, Link } from 'react-router-dom'
 import client from '../../utils/client'
 import Header from '../Header/Header'
 
 import './styles/ProfilePage.css'
 
 function ProfilePage({ loggedInUser }) {
-  const [currentUser, setCurrentUser] = useState(null)
+  const [profilePageUser, setprofilePageUser] = useState(null)
```
Comment (Auenc) [inline src/components/profile/ProfilePage.js:9]: `setprofilePageUser` should be `setProfilePageUser`

File: src/components/profile/ProfilePage.js [hunk 4]
```diff
@@ -1,61 +1,73 @@
...
     // TODO: Change loading state here
   }, [id])
 
-  if (currentUser === null) {
+  if (profilePageUser === null) {
     return <p> Loading...</p>
   }
 
-  if (currentUser !== null) {
+  if (profilePageUser !== null) {
```
Comment (Auenc) [inline src/components/profile/ProfilePage.js:25]: Let's refactor this to instead do 

```js
if(profilePageUser === null) {
    return <></>
}

return (
...
)
```

**Review by Auenc — CHANGES_REQUESTED** [review]

File: button.css [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  appearance: none;
+  background-color: #FAFBFC;
+  border: 1px solid rgba(27, 31, 35, 0.15);
+  border-radius: 6px;
+  box-shadow: rgba(27, 31, 35, 0.04) 0 1px 0, rgba(255, 255, 255, 0.25) 0 1px 0 inset;
+  box-sizing: border-box;
+  color: #24292E;
+  cursor: pointer;
+  display: inline-block;
+  font-family: -apple-system, system-ui, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
```
Comment (chris2733) [inline button.css:15]: Be wary of repeating lines like this - you could instead have base styles for a button, then add modifiers to this class for other styles, happy to share more information on this if needed

**Review by Auenc — APPROVED** [review]

**Pull request merged**